integrations/slack/webhooks/webhook.go

```diff
@@ -195,3 +234,46 @@ func (h handler) dispatchAsyncEventsToConnections(ctx context.Context, cids []sd
 		l.Debug("Event dispatched")
 	}
 }
+
+func (h handler) extractIDs(body []byte, wantContentType string, l *zap.Logger) (appID, teamID, enterpriseID string, err error) {
+	if wantContentType == "application/json" {
+		var cb events.Callback
+		if err := json.Unmarshal(body, &cb); err != nil {
+			l.Error("Failed to parse JSON for app/team IDs",
+				zap.Error(err),
+			)
+			return "", "", "", err
+		}
+		return cb.APIAppID, cb.TeamID, "", nil
```

Comment:
Add a TODO comment to handle enterprises instead of return. No need for a Linear ticket because I don't know if'll ever get to this, unless a specific user/customer requires it.